### PR TITLE
Fix erb comment format

### PR DESCRIPTION
**Pull request by Focinfi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -919,13 +919,13 @@ your application.
   methods don't.
 <sup>[[link](#url-not-path-in-email)]</sup>
 
-  ```Ruby
-  # bad
-  You can always find more info about this course
+  ```ERB
+  <%# bad %>
```
Comment (bbatsov) [inline README.md]: Add a space between % and #

**Comment by bbatsov** [comment]
Please, change the commit message to `Fix erb comment format`.

**Comment by KevinBongart** [comment]
Uh… what's the point of this PR? I'm not sure I understand

**Comment by onebree** [comment]
@KevinBongart I think because since the code block would lie inside an ERB file, it is best for the guide to present that information. Also, it shows how to effectively make Ruby-style comments in ERB.

**Comment by philoserf** [comment]
I do not believe this pull request fixes an error. Reading the existing example, I believe we have it right as is.

**Comment by KevinBongart** [comment]
Yeah that's what I was thinking: while these changes are technically correct (I didn't know GitHub supported ERB syntax coloring), the rest of the file seem to be using standard Ruby comments

**Comment by philoserf** [comment]
The #bad isn't really a comment. It is a market in the guide.

The line of text is not really a comment. It is content preceding the <%=.

**Comment by bbatsov** [comment]
OK, guess we'll leave this as is, then. After all the meaning of the example is pretty clear even now.

**Pull request closed without merging**